src/PJ_isea.c

```diff
@@ -309,8 +291,7 @@ sph_azimuth(double f_lon, double f_lat, double t_lon, double t_lat)
 #endif
 
 /* coord needs to be in radians */
-ISEA_STATIC
-int
+static int
```

Comment:
A small style nit here. In some cases the `static int` (or whatever) is on a line by itself, in others it is on the same line as the function. I prefer the latter but at least it should be consistent across the file.

Reply:
Done.  I was trying not to worry to much about formatting without the project having an official style (or better yet a clang-format definition I can just run before committing).

Reply:
I know. Thanks for doing the touch up. About the style guide... we'll get there eventually :-)